lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -259,22 +365,22 @@ class MockTimers {
     this.#isEnabled = activate;
   }
 
-  tick(time = 1) {
-    if (!this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'You should enable MockTimers first by calling the .enable function',
-      );
+  #isValidDateWithGetTime(maybeDate) {
+    // Validation inspired on https://github.com/inspect-js/is-date-object/blob/main/index.js#L3-L11
+    const getTime = this.#realDate.prototype.getTime;
+    try {
+      getTime.call(maybeDate);
```

Comment:
we can't rely on Function.prototype.call being present; you can use `DatePrototypeGetTime` here instead
```suggestion
    try {
      DatePrototypeGetTime(maybeDate);
```